src/backend/logger/logger.ts

```diff
@@ -468,6 +474,128 @@ class GameLogWriter extends LogWriter {
     this.filePath = lastPlayLogFileLocation(gameInfo.app_name)
   }
 
+  // cleanup settings for logs to avoid confusions during support requests
+  filterGameSettingsForLog(
+    gameSettings: Partial<GameSettings>,
+    notNative: boolean
+  ): Partial<GameSettings> {
+    // remove gamescope settings if it's disabled
+    const gscope = gameSettings['gamescope'] as Partial<GameScopeSettings>
```

Comment:
```suggestion
    const gscope: Partial<GameScopeSettings> | undefined = gameSettings['gamescope']
```
This may seem a little excessive, but *always* avoid `as` if you can do it somehow.  
Why? Try changing `'gamescope'` to `'someOtherNonexistantName'`. No errors! Alternatively, try changing the `GameSettings` type, changing or removing `gamescope`. Again, no error here